src/modules/storefront/components/StorefrontPreview.tsx

```diff
@@ -84,20 +88,38 @@ const Storefront = ({ params, children }: StorefrontPreviewProps) => {
   const data = { ...context?.pageTheme, ...currentPage };
   const themeContext = context?.defaultTheme;
 
+  const breakpoint = useBreakpoints();
+  const mobileBreakpoints = [breakpointsEnum.SM, breakpointsEnum.XS];
+
   if (!themeContext) return null;
   const isProductPage =
     (asPath || '').includes('/product/slug') &&
     params?.[params?.length - 1] != 'slug';
   const theme = { ...context.defaultTheme, ...themeListener };
-  const fontName = theme.configurations?.styleData.fontFamily;
+
+  const configStyleData = theme.configurations?.styleData;
+  const configMobileStyleData = theme.configurations?.mobileStyleData;
+
+  const mergedConfigStyleData = mobileBreakpoints.includes(breakpoint)
+    ? { ...configStyleData, ...configMobileStyleData }
+    : configStyleData;
```

Comment:
use a função que você criou.

Reply:
nesse caso não dá, por causa do `if (!themeContext) return null;` na linha 94, e os hooks só podem ser usados antes do if, porque eles precisam manter a consistencia entre uma renderização e outra. se o themeContext for vazio ele "renderiza null"